diff --git a/app/layout.tsx b/app/layout.tsx
index 5087aa5..e96d637 100644
--- a/app/layout.tsx
+++ b/app/layout.tsx
@@ -1,30 +1,44 @@
-import { Geist, Geist_Mono } from "next/font/google"
+import { Geist, Geist_Mono } from "next/font/google";
 
-import "./globals.css"
-import { ThemeProvider } from "@/components/theme-provider"
+import "./globals.css";
+import { ThemeProvider } from "@/components/theme-provider";
 import { cn } from "@/lib/utils";
+import { Toaster } from "@/components/ui/sonner";
+import { Navbar } from "@/components/navbar";
+import { PageAIContextProvider } from "@btst/stack/plugins/ai-chat/client/context";
 
-const geist = Geist({subsets:['latin'],variable:'--font-sans'})
+const geist = Geist({ subsets: ["latin"], variable: "--font-sans" });
 
 const fontMono = Geist_Mono({
-  subsets: ["latin"],
-  variable: "--font-mono",
-})
+	subsets: ["latin"],
+	variable: "--font-mono",
+});
 
 export default function RootLayout({
-  children,
+	children,
 }: Readonly<{
-  children: React.ReactNode
+	children: React.ReactNode;
 }>) {
-  return (
-    <html
-      lang="en"
-      suppressHydrationWarning
-      className={cn("antialiased", fontMono.variable, "font-sans", geist.variable)}
-    >
-      <body>
-        <ThemeProvider>{children}</ThemeProvider>
-      </body>
-    </html>
-  )
+	return (
+		<html
+			lang="en"
+			suppressHydrationWarning
+			className={cn(
+				"antialiased",
+				fontMono.variable,
+				"font-sans",
+				geist.variable,
+			)}
+		>
+			<body>
+				<PageAIContextProvider>
+					<ThemeProvider>
+						<Navbar />
+						{children}
+						<Toaster />
+					</ThemeProvider>
+				</PageAIContextProvider>
+			</body>
+		</html>
+	);
 }
