diff --git a/components/mode-toggle.tsx b/components/mode-toggle.tsx
index a7936e4..8679f76 100644
--- a/components/mode-toggle.tsx
+++ b/components/mode-toggle.tsx
@@ -1,17 +1,17 @@
-"use client"
+"use client";
 
-import { MonitorIcon, MoonIcon, SunIcon } from "lucide-react"
-import { useTheme } from "next-themes"
-import { Button } from "@/components/ui/button"
+import { MonitorIcon, MoonIcon, SunIcon } from "lucide-react";
+import { useTheme } from "next-themes";
+import { Button } from "@/components/ui/button";
 import {
 	DropdownMenu,
 	DropdownMenuContent,
 	DropdownMenuItem,
 	DropdownMenuTrigger,
-} from "@/components/ui/dropdown-menu"
+} from "@/components/ui/dropdown-menu";
 
 export function ModeToggle() {
-	const { setTheme } = useTheme()
+	const { setTheme } = useTheme();
 
 	return (
 		<DropdownMenu>
@@ -22,7 +22,10 @@ export function ModeToggle() {
 					<span className="sr-only">Toggle theme</span>
 				</Button>
 			</DropdownMenuTrigger>
-			<DropdownMenuContent align="end" onCloseAutoFocus={(e) => e.preventDefault()}>
+			<DropdownMenuContent
+				align="end"
+				onCloseAutoFocus={(e) => e.preventDefault()}
+			>
 				<DropdownMenuItem onClick={() => setTheme("light")}>
 					<SunIcon />
 					Light
@@ -37,5 +40,5 @@ export function ModeToggle() {
 				</DropdownMenuItem>
 			</DropdownMenuContent>
 		</DropdownMenu>
-	)
+	);
 }
